src/environments/environment.prod.ts

```diff
@@ -4,5 +4,6 @@ export const environment = {
     
   },
   useEmulators: false,
-  gitApiUrl: "https://api.github.com/repos/Worktez/worktez"
+  gitApiUrl: "https://api.github.com/repos/Worktez/worktez",
```

Comment:
Add the new variable in normal key as well

Reply:
done